.circleci/config.yml

```
@@ -1573,15 +1573,14 @@ jobs:
 
   test-e2e-react:
     docker:
-      - image: mcr.microsoft.com/playwright:v1.33.0-focal
+      - image: cimg/node:20.1.0
     resource_class: large
     steps:
       - checkout-with-sm
       - skip-if-docs-only
       - skip-if-github-only
-      - run: apt-get update && apt-get -y install make
       - react-get-deps
-      - run: npx playwright install --with-deps && npx playwright install chrome
+      - run: npx playwright install --with-deps chrome firefox webkit
```

playwright should have webkit and firefox installed already. we should only need chrome installed because it installs outside of the cache. We could try to cut down run times by only installing the deps and chrome.
```suggestion
      - run: npx playwright --with-deps chrome
```